### ISSUE TITLE
Patchwork.json issue

### ISSUE THREAD

**Issue by seandreea** [antecedent/patchwork#30]
After upgrading to Patchwork 1.4, my team is having issues running Patchwork.
Upon investigation I found 2 issues:
1) In vendor/antecedent/patchwork/src/Config.php, line 18, it seems to be searching for patchwork.json in the root patchwork folder, but that file is present in antecedent/patchwork/tests folder. It is called from vendor/antecedent/patchwork/Patchwork.php, line 93: Config\locate(**DIR**); . I copied the file in the root folder and it seems ok, but this is not a good long term solution. 
2) Couchbase functions are seen an user-defined ("couchbase_basic_encoder_v1") for example. But these are from the PHP Extension, so they should be included before runtime, right? I know this is not a Patchwork issue, but I was wondering if you had any idea where we could place the require Patchwork code, in order to overcome this? (it is currently placed at the very top of the PHPUnit bootstrap file)

**Comment by antecedent** [comment]
At that point of time, Patchwork 1.4 was yet to be released. Sorry about the misleading README.

1. As of now, `patchwork.json` is optional, and, if present, will be picked from the entry script's directory or any directory enclosing it.
2. Try suppressing the warning with `@`.

**Comment by seandreea** [comment]
Hi,

About the  warning: that is what we did, but I was wondering if there was a more elegant way around this.
As for the first issue, it seems that the script enters and endless loop and doesn't get past this point. As I mentioned, I tried moving patchwork.json in the root folder and it worked, but I hope it can be fixed in the following versions.

**Comment by antecedent** [comment]
Please try the latest commit. The endless looping should be fixed, and `patchwork.json` now has an `ignore` key, which you could use as follows:

```json
{
    "ignore": [
        "couchbase_*",
        "cbdsn_*",
        "_cbdsn_*",
        "Couchbase*",
        "_Couchbase*"
    ]
}
```

**Comment by antecedent** [comment]
Update: I've finally released Patchwork 1.4, which should now automatically suppress the warnings for PHPUnit- and Couchbase-related methods.
